rust-runtime/inlineable/src/event_stream_receiver.rs

```diff
@@ -0,0 +1,30 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+use aws_smithy_http::event_stream::Receiver;
+use aws_smithy_runtime_api::box_error::BoxError;
+
+#[derive(Debug)]
+#[non_exhaustive]
+/// Receives messages out of an Event Stream.
+pub struct EventStreamReceiver<T, E> {
```

Comment:
What do you think about calling this `EventReceiver` since it receives individual events at a time rather than an entire stream at a time?

Reply:
> since it receives individual events at a time

I like that the suggested name implies that. Updated the name in b3f4b86.